wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/spi/InternalRuntime.java

```diff
@@ -60,7 +70,12 @@ public String getActiveEnv() {
 
   @Override
   public String getOwner() {
-    return context.getIdentity().getOwnerName();
+    try {
+      User user = userDao.getById(context.getIdentity().getOwnerId());
```

Comment:
I don't think it is good to return owner name or owner id here. Let's document that owner id will be returned in `Runtime` class and just return it.

Reply:
> Maybe we could just throw an exception instead.

You can try to do it but I think it might be difficult with current classes hierarchy

> Change the meaning of value from a plain user name to an ID (opaque to the user), in a location that is used in many places (even in the RuntimeDto).

Makes sense. But I think using username there is a mistake that was done because of docs lack.
@gazarenkov should be right person who knows what is supposed to be there.

If user id then I think we should try to change it and run QA tests. I guess clients don't use it and everything will be OK :wink:

Reply:
I think we agreed in your previous PR that we should do this change. I don't think that clients use it somehow now. But running tests, indeed, should verify this.